data/mods/ssb/moves.ts

```diff
@@ -853,7 +853,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		onPrepareHit(target, source) {
 			this.add('-anim', source, 'Spirit Break', target);
 		},
-		useSourceDefensiveAsOffensive: true,
+		useBaseOffensiveStatAndBoosts: 'source:spd',
```

Comment:
this would probably be better as an array or something `['source', 'spd']` for example

Reply:
agreed. Don't know why I, and none of the people on the discord server who talked about this, never considered just making it an array. We treat it as one anyway. Good catch.

Reply:
Those slip-ups happen sometimes, and it functions the same regardless, so it probably wasn't at the top of everyone's minds when the main focus was making sure it worked.